skimage/segmentation/slic_superpixels.py

```diff
@@ -136,9 +136,7 @@ def slic(image, n_segments=100, compactness=10., max_iter=10, sigma=0,
     elif isinstance(sigma, (list, tuple)):
         sigma = np.array(sigma, dtype=np.double)
     if (sigma > 0).any():
-        # add zero smoothing for multichannel dimension
-        sigma = list(sigma) + [0]
-        image = ndi.gaussian_filter(image, sigma)
+        image = gaussian(image, sigma, multichannel=True)
```

Comment:
As an aside, @jni, is this working as expected? It seems there are a lot of conditionals and this function should support a wide array of inputs... but the only Gaussian smoothing in the whole function is here, in this conditional, and it was hardcoded to ignore the final dimension. 

My change does do the exact same thing - but something seems off.